geanypg/configure.ac

```diff
@@ -8,7 +8,7 @@ AC_PROG_LIBTOOL
 AM_PATH_GPGME()
 
 # checking for Geany
-PKG_CHECK_MODULES(GEANY, [geany >= 0.20])
+PKG_CHECK_MODULES(GEANY, [geany >= 1.24])
```

Comment:
why?  no changes here seem to suggest the plugin needs a more recent Geany version.